dropwizard-logging/src/main/java/io/dropwizard/logging/FileAppenderFactory.java

```diff
@@ -105,8 +105,8 @@
  */
 @JsonTypeName("file")
 public class FileAppenderFactory<E extends DeferredProcessingAware> extends AbstractAppenderFactory<E> {
-    @NotNull
-    private String currentLogFilename;
+
+	private String currentLogFilename;
```

Comment:
Looks like you used a tab instead of spaces here. This issue also occurs in the tests. Please fix.